utils/das_data_sorter.py

```diff
@@ -1,76 +1,170 @@
 import pandas as pd 
-import argparse 
-from math import ceil
+from argparse import ArgumentParser
+from numpy import ceil
+from typing import TypeVar
 
-# averages data within the seconds, according to seconds array 
-def avg_Data(array):
-    avg_array = []
+D = TypeVar('DAS_data') # data type to represent columns of the data in a given file
 
-    for index_array in new_time:
-        total = 0
-        for index in index_array:
-            total += array[index]
-
-        avg = round(total/len(index_array), ndigits=2)
-        avg_array.append(avg)
-    return avg_array
-
-# accepts terminal commands
-parser = argparse.ArgumentParser()
+# accepts terminal arguments
+parser = ArgumentParser()
 parser.add_argument("--file", help="Input CSV file", action="store", required=True)
 parser.add_argument("--output", help="Returns the filtered data", default="filtered_data.csv", action="store")
 parser.add_argument("--unit", help="Specify time units (seconds, s, or minutes, m)", default="seconds", 
                     choices=["seconds", "s", "minutes", "m"], action="store")
```

Comment:
I didn't make these required because I have put a default argument for these. 
Output default = `filtered_data.csv`
Unit default = `seconds`
Is this fine or change it?

Reply:
Actually - I don't think we should have a default output, as it could lead to accidentally overwriting a file. We should either have no default, or have some confirmation about whether you really want to overwrite. (The former would be much easier)